src/components/datepicker/js/datepickerDirective.js

```diff
@@ -338,6 +330,25 @@
     /** @type {Number} Extra margin for the top of the floating calendar. Gets determined on the first open. */
     this.topMargin = null;
 
+    /** @type {!MdPanelPosition} Position object for the mdPanel instance. */
+    this.panelPosition = $mdPanel.newPanelPosition()
+      .relativeTo(this.inputContainer)
+      .addPanelPosition($mdPanel.xPosition.ALIGN_START, $mdPanel.yPosition.ALIGN_TOPS)
+      .withOffsetX(-this.leftMargin + 'px')
+      .withOffsetY(angular.bind(this, this.getPanelYOffset));
+
```

Comment:
The current version of the md-datepicker hides the popup when the window is re-sized. I noticed that the md-panel stays in the original position when the window is resized and using "relativeTo" (it stops being relativeTo the original element).

Can you confirm what the behavior is for the new implementation?